From a7f57ad9465811494d1070feb4f2407b1a57c537 Mon Sep 17 00:00:00 2001
From: Mikel Olasagasti Uranga <mikel@olasagasti.info>
Date: Wed, 1 Sep 2021 22:46:34 +0200
Subject: [PATCH] Skip network test

---
 file_test.go | 1 +
 1 file changed, 1 insertion(+)

diff --git a/file_test.go b/file_test.go
index 238c267..ceb3527 100644
--- a/file_test.go
+++ b/file_test.go
@@ -66,6 +66,7 @@ func TestDeleteFilesOlderThan(t *testing.T) {
 }
 
 func TestDownloadFile(t *testing.T) {
+	t.Skip("Skip test as network is inaccessible during package build")
 	// attempt to download http://ipv4.download.thinkbroadband.com/5MB.zip to temp folder
 	tmpfile, err := os.CreateTemp("", "")
 	require.Nil(t, err, "couldn't create folder: %s", err)
-- 
2.31.1

